package.json

```diff
@@ -1,90 +1,95 @@
-    "@storybook/addon-essentials": "^6.4.9",
-    "@storybook/addon-links": "^6.4.9",
-    "@storybook/builder-webpack5": "^6.4.9",
-    "@storybook/manager-webpack5": "^6.4.9",
-    "@storybook/react": "^6.4.9",
-    "@svgr/cli": "^6.1.1",
-    "concurrently": "^6.4.0",
-    "nodemon": "^2.0.15",
-    "react-calendar": "^3.5.0"
-  },
-  "dependencies": {
-    "@babel/core": "^7.16.0",
-    "@babel/plugin-proposal-class-properties": "^7.16.0",
-    "@babel/plugin-proposal-object-rest-spread": "^7.16.0",
-    "@babel/plugin-proposal-private-property-in-object": "^7.16.0",
-    "@babel/plugin-transform-react-inline-elements": "^7.16.0",
-    "@babel/plugin-transform-regenerator": "^7.16.0",
-    "@babel/plugin-transform-runtime": "^7.16.4",
-    "@babel/preset-env": "^7.16.4",
-    "@babel/preset-react": "^7.16.0",
-    "@babel/runtime": "^7.16.3",
-    "babel-loader": "^8.2.3",
-    "civil-client": "github:EnCiv/civil-client",
-    "civil-server": "github:EnCiv/civil-server#main",
-    "classnames": "^2.3.1",
-    "humanize-string": "^3.0.0",
-    "isemail": "^3.2.0",
-    "isomorphic-mongo-objectid": "^1.0.9",
-    "moment": "^2.29.1",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
-    "react-hot-loader": "^4.13.0",
-    "react-jss": "^10.9.0",
-    "react-textarea-autosize": "^8.3.3",
-    "undebate": "github:EnCiv/undebate#civ-serv",
-    "webpack": "^5.65.0",
-    "webpack-cli": "^4.9.1",
-    "webpack-dev-server": "^4.6.0"
-  },
-  "engines": {
-    "node": "~14.x.x",
-    "npm": "~7.17.x"
-  }
+    "name": "undebate-ssp",
+    "version": "v0.0.0",
+    "description": "Undebate Self Service Portal",
+    "main": "index.js",
+    "scripts": {
+        "db-up": "start /b mongod --quiet --dbpath ./tmp/db --port 27017",
+        "db-down": "mongo --eval \"db.getSiblingDB('admin').shutdownServer(); quit()\"",
+        "dev": "echo \"these delays need to be kept up to date\" && concurrently -k \"npm run hot-transpile\" \"sleep 20 && npm run hot-server\" \"sleep 15 && npm run hot-client\"",
+        "hot-client": "webpack-dev-server --config webpack-dev.config.js --devtool source-map --host 0.0.0.0",
+        "hot-server": "nodemon --inspect dist/start.js",
+        "hot-transpile": "babel --watch app --ignore **/__tests__ --out-dir dist --source-maps",
+        "packbuild": "webpack --config webpack-prod.config.js",
+        "postinstall": "bash postinstall.sh",
+        "prestart": "npm run packbuild",
+        "start": "node --max_old_space_size=460 dist/start.js",
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "transpile": "babel app --ignore **/__tests__ --out-dir dist --source-maps",
+        "storybook": "start-storybook -p 6006",
+        "build-storybook": "build-storybook",
+        "svgr": "svgr --icon --filename-case kebab assets/svg --out-dir app/svgr && rm app/svgr/index.js",
+        "format": "prettier --write \"./**/*.{js,jsx,css,scss,md,json}\" --config ./.prettierrc",
```

Comment:
for "format", it didn't work on my machine.  the code was using single quotes to wrap the write parameter, instead I used the \" which is also used elsewhere.  Please try this in your environment and let me know if it works or if there is a problem.